bin/pytorch_inference/Main.cc

```diff
@@ -21,10 +20,14 @@
 #include "CCmdLineParser.h"
 #include "CCommandParser.h"
 
+#include <ATen/Parallel.h>
 #include <rapidjson/ostreamwrapper.h>
+#include <torch/csrc/api/include/torch/types.h>
 #include <torch/script.h>
 
+#include <cstdint>
 #include <memory>
+#include <ostream>
```

Comment:
It doesn't look like any of the additions should need this.

Reply:
Hmm VSCode must have added it